### DISCUSSION TITLE
Structure of qrg.txt file ?

### DISCUSSION THREAD

**Discussion by tk5ep** [dl9rdz/rdz_ttgo_sonde#294] [Q&A] [answered]
HI all,

Can someone give me the structure of the QRG.TXT file ?
What is the number for the M10/M20 ?
For what are the + / - sign ?

Update : Meanwhile, I found that the +/- are to enable and disable the frequency.

Thanks and 73
Patrick TK5EP

**Comment by LY3OO** [comment, marked as answer]
+ / - is Active/Inactive status
![image](https://user-images.githubusercontent.com/102389281/163730244-0dd7c6e0-17b1-498a-bf65-b83fa9d4cc6f.png)

stypes.set('4', 'RS41');
stypes.set('R', 'RS92');
stypes.set('D', 'DFM');
stypes.set('M', 'M10/M20');
//stypes.set('2', 'M10/M20');
stypes.set('3', 'MP3H');

73 de LY3OO

**Reply by tk5ep** [reply]
Thanks a lot ! 
It works with these values. (not described in the qrg.txt example file )

73 de TK5EP

**Comment by dl9rdz** [comment]
I have updated the documentation in qrg.txt just in case someone needs this...

**Reply by tk5ep** [reply]
Thanks and 73
